backend/aws_helpers/dynamo_db_utils/trainspace_db.py

```diff
@@ -27,7 +27,7 @@ class TrainspaceData:
     name: str = ""
     parameters_data: dict = None
     review_data: str = ""
-    status: TrainStatus = TrainStatus.QUEUED
+    # status: TrainStatus = TrainStatus.QUEUED
```

Comment:
why is this commented out?

Reply:
shouldn't we need to track trainspace status to know if a train is queued, training, succeeded, failed, etc?

Reply:
Yes, but dynamo doesn't understand enums, so I just commented that out just to not deal with that issue rn. When we refactor, we could make a Tabular class and add a to_dynamo method that deals with both solving the decimal issue and this issue also